src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -66,7 +66,7 @@
         <TargetPath>tools</TargetPath>
       </RuntimeFiles>
 
-      <RuntimeFiles Condition="'$(TargetsMobile)' == 'true'"
+      <RuntimeFiles
```

Comment:
Would this include `MonoIncludeFiles` for all platforms? Is that desired, or is it somehow only including it in `RuntimeFiles` for mono platforms? Should we use `'$(RuntimeFlavor)' == 'mono'` to make sure its only for mono platforms?

Reply:
It would include it for all platforms, and you're right that it would include CoreCLR in theory - but since it's only including `@(MonoIncludeFiles)` which would be empty on CoreCLR, we should be safe. If you feel it's worth adding the conditional anyway, I guess we can do that